src/components/Clock/Clock.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Clock extends React.Component {
+  state = {
+    time: new Date(),
+  };
+
+  timer = setInterval(() => {
+    this.setState({
+      time: new Date(),
+    });
+
+    // eslint-disable-next-line
+    console.log(new Date().toLocaleTimeString());
+  }, 1000);
+
+  componentDidUpdate(prevProps) {
+    if (this.props.timerId !== prevProps.timerId) {
+      // eslint-disable-next-line
+      console.log(`
+        The timer was renamed from ${prevProps.timerId} to ${this.props.timerId}
+      `);
+    }
+  }
```

Comment:
It's great that you have used componentDidUpdate, but for this task, you should only console.log the new name only when clicking the button (so you could easily put the 'console.log to your 'changeId' method)